src/main/webapp/app/shared/user-settings/notification-settings/notification-settings.default.ts

```diff
@@ -0,0 +1,104 @@
+import { Authority } from 'app/shared/constants/authority.constants';
+import { OptionSpecifier, UserSettingsCategory } from 'app/shared/constants/user-settings.constants';
+import { OptionCore, UserSettings } from '../user-settings.model';
+
+export interface NotificationOptionCore extends OptionCore {
+    webapp: boolean;
+    email?: boolean;
+}
+
+export const defaultNotificationSettings: UserSettings<NotificationOptionCore> = {
+    category: UserSettingsCategory.NOTIFICATION_SETTINGS,
+    groups: [
+        {
+            name: 'Exercise Notifications',
+            restrictionLevel: Authority.USER,
+            options: [
+                {
+                    name: 'Exercise created or started',
+                    description: 'Get notified if a new exercise has been created or started',
```

Comment:
why can't we use actual translation keys, such as "ExerciseCreatedOrStarted", I would change this in combination with my comment [here](https://github.com/ls1intum/Artemis/pull/3922/files#r710303615). Or am I missing something completely?

Reply:
Please have a look at my response above